socketio-client.gemspec

```
@@ -19,5 +19,6 @@ Gem::Specification.new do |s|
 
   # specify any dependencies here; for example:
   # s.add_development_dependency "rspec"
-  s.add_dependency 'rest-client', '~> 1.8.0'
+  s.add_dependency 'rest-client', '~> 2.0.1'
+  s.add_dependency 'byebug'
```

```suggestion
  s.add_development_dependency 'byebug'
```

It'd probably be best if we kept this as part of this repo, and not impose it on all of socketio-client's consumers :)